### PR TITLE
[ADD] l10n_pe: add Spanish translations

### PR DESCRIPTION
**Pull request by umiphos**
Description of the issue/feature this PR addresses:

Adding the spanish translation for l10n_pe and the POT for later translations. 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_pe/i18n/l10n_pe.pot [hunk 1]
```diff
@@ -0,0 +1,9611 @@
...
+msgid ""
+"Follow the UN/ECE 5305 standard from the United Nations Economic Commission "
+"for Europe for more information "
+"http://www.unece.org/trade/untdid/d08a/tred/tred5305.htm"
+msgstr ""
+
+#. module: l10n_pe
+#: model:ir.model.fields.selection,name:l10n_pe.selection__account_tax__l10n_pe_edi_unece_category__g
+#: model:ir.model.fields.selection,name:l10n_pe.selection__account_tax_template__l10n_pe_edi_unece_category__g
+msgid "Free export item, tax not charged"
```
Comment (ren-odoo) [inline addons/l10n_pe/i18n/l10n_pe.pot:2892]: @umiphos Is there any reason why these terms are not translated?

File: addons/l10n_pe/i18n/l10n_pe.pot [hunk 2]
```diff
@@ -0,0 +1,9611 @@
...
+
+#. module: l10n_pe
+#: model:account.tax.group,name:l10n_pe.tax_group_gra
+msgid "GRA"
+msgstr ""
+
+#. module: l10n_pe
+#: model:ir.model.fields.selection,name:l10n_pe.selection__account_tax__l10n_pe_edi_tax_code__9996
+#: model:ir.model.fields.selection,name:l10n_pe.selection__account_tax_template__l10n_pe_edi_tax_code__9996
+msgid "GRA - Free"
```
Comment (ren-odoo) [inline addons/l10n_pe/i18n/l10n_pe.pot:2908]: @umiphos same comment than before.

Comment (umiphos) [reply addons/l10n_pe/i18n/l10n_pe.pot:2908]: the POT file is not translated it's only a template file for translations, in the es.po file is the real spanish translation

**Comment by ren-odoo** [comment]
Hi @jco-odoo 

This PR is ready for review. Thank you. 

cc. @vbe-odoo

File: addons/l10n_pe/i18n/l10n_pe.pot [hunk 3]
```diff
@@ -0,0 +1,9611 @@
...
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_010311
+msgid "Valera"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_160606
+msgid "Vargas Guerra"
```
Comment (jco-odoo) [inline addons/l10n_pe/i18n/l10n_pe.pot:9132]: Probably districts should not be translatable.

**Review by jco-odoo — COMMENTED** [review]
@umiphos There are probably 18000 lines for no reason. @ren-odoo Not sure what to do with it.  @mart-e  Can we just remove them from the po/pot?

File: addons/l10n_pe/i18n/l10n_pe.pot (hunk 3 above)
Comment (umiphos) [reply addons/l10n_pe/i18n/l10n_pe.pot:9132]: Removing all translations that will not be translated like districts, first PR was following the logic of exporting the POT like in l10n_mx_edi
https://github.com/odoo/enterprise/blob/14.0/l10n_mx_edi/i18n/l10n_mx_edi.pot

**Comment by mart-e** [comment]
Hi,

I agree with Josse, there is 38 translations out of a 1854 translations in `es.po`. Could you keep only the relevant one.
```
$ msgattrib --translated addons/l10n_pe/i18n/es.po -o addons/l10n_pe/i18n/es.po
```
should do the trick

**Comment by ren-odoo** [comment]
> uld you keep only the relevant one.

Hi @umiphos can you confirmed if this comment has been addressed already?

Thank you .

**Comment by umiphos** [comment]
@ren-odoo I just updated the brach, this is ready to review again. 
cc @jco-odoo

**Comment by jco-odoo** [comment]
Thanks @umiphos

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**